ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/TableViewCell/ic_fluent_lock_closed_20_regular.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "ic_fluent_lock_closed_20_regular.pdf",
```

Comment:
hmmm i would have expected there will be one of this. what is the asset setting on the image? single scale?

Reply:
like there should n't be 1x, 2, 3x in the json file

Reply:
Yep, I'll remove them.